runtime/runtime/src/prefetch.rs

```diff
@@ -304,6 +328,73 @@ impl TriePrefetcher {
         Ok(())
     }
 
+    /// Prefetcher tuned for claim.sweat::claim contract calls.
+    ///
+    /// Remove after #10965 reaches mainnet.
+    fn prefetch_claim_sweat_claim(
+        &self,
+        account_id: AccountId,
+        predecessor: AccountId,
+    ) -> Result<(), PrefetchError> {
+        let Self { prefetch_api, trie_root, .. } = self;
+        let trie_root = *trie_root;
+        let prefetch_api = prefetch_api.clone();
+        rayon::spawn(move || {
+            let mut account_data_key = Vec::with_capacity(4 + 8 + predecessor.len());
+            let Ok(()) = 0u8.serialize(&mut account_data_key) else { return };
+            let Ok(()) = predecessor.serialize(&mut account_data_key) else { return };
+            let trie_key =
+                TrieKey::ContractData { account_id: account_id.clone(), key: account_data_key };
+            // Just read this directly for now since this is temporary anyway
+            let prefetcher_storage = prefetch_api.make_storage();
+            let trie = Trie::new(prefetcher_storage, trie_root, None);
+            let Ok(Some(account_record)) = trie.get(&trie_key.to_vec()) else {
+                tracing::debug!(
+                    target: "runtime::prefetch",
+                    message = "could not load AccountRecord",
+                    key = ?trie_key,
+                );
+                return;
+            };
+            #[derive(borsh::BorshDeserialize)]
+            #[allow(dead_code)]
+            struct AccountRecord {
+                accruals: Vec<(u32, u32)>,
+                is_enabled: bool,
+                claim_period_refreshed_at: u32,
+                is_locked: bool,
+            }
+            let Ok(account_record) = borsh::from_slice::<AccountRecord>(&account_record) else {
+                tracing::debug!(
+                    target: "runtime::prefetch",
+                    message = "could not decode AccountRecord",
+                );
+                return;
+            };
+
+            for (dt, idx) in account_record.accruals {
```

Comment:
Idea: if we think there could be especially large number of `accruals` in some of the records, it might make sense to limit the work in this thread to only prefetch the first K of them.

Reply:
How do we know what limit to use? But also, given that this code is going to be dead in 3 weeks...